### PR TITLE
Add @bisaloo as an author and @pratikunterwegs as a contributor

### PR DESCRIPTION
**Pull request by seabbs**
This PR adds @Bisaloo as a package author and @pratikunterwegs as a contributor and reorders the DESCRIPTION to go authors and then contributors. I may have missed people who should be credited here so please make suggestions.

@Bisaloo and @pratikunterwegs if you could both confirm you are fine to be added here that would be great.

### CODE REVIEW

**Review by pearsonca — APPROVED** [review]
one consistency note, but technically fine

File: DESCRIPTION [hunk 1]
```diff
@@ -15,6 +15,16 @@ Authors@R:
...
            email = "sebastian.funk@lshtm.ac.uk"),
+    person(given = "Carl",
+           family = "Pearson",
+           role = "aut",
+           email = "carl.ab.pearson@gmail.com",
+           comment = c(ORCID = "0000-0003-0701-7860")),
+    person(given = "Hugo",
+           family = "Gruson",
+           role = c("aut"),
+           email = "hugo.gruson@lshtm.ac.uk",
```
Comment (Bisaloo) [inline DESCRIPTION:26]:
```suggestion
           email = "hugo.gruson@lshtm.ac.uk",
```

This address was deleted the second I left LSHTM. Could you please replace it by hugo.gruson@normalesup.org or just remove it?

File: DESCRIPTION [hunk 2]
```diff
@@ -15,6 +15,16 @@ Authors@R:
...
            role = "aut",
            email = "sebastian.funk@lshtm.ac.uk"),
+    person(given = "Carl",
+           family = "Pearson",
+           role = "aut",
+           email = "carl.ab.pearson@gmail.com",
+           comment = c(ORCID = "0000-0003-0701-7860")),
+    person(given = "Hugo",
+           family = "Gruson",
+           role = c("aut"),
```
[the remark this answers is not in the document]
Comment (seabbs) [reply DESCRIPTION:25]: Ah yes good spot. I can't amend as only have my phone but will make another PR fixing this soon (or you are welcome to 😎)

**Review by adrian-lison — COMMENTED** [review]
Great to have new contributors/authors :)

**Pull request merged**